### ISSUE TITLE
Document `imbaconfig.json` and its theming capabilities

### ISSUE THREAD

**Issue by andregoldstein** [imba/imba.io#196]
https://discord.com/channels/682180555286380545/682180555286380726/877896113947033610

> You can add a shared file where you define some css variables @root with your colors, and then refer to those variables, but it is also possible to declare your own color palettes in imbaconfig.json - it is just not documented :stuck_out_tongue:

**Comment by haikyuu** [comment]
Yes, we can also create custom esbuild loaders there. And we can write that file in Imba as well ...
I don't know how 😆
